default.vcl

```diff
@@ -354,8 +354,8 @@ sub vcl_recv {
     } else if (req.url ~ "^\/concept\/lists.*$") {
             set req.url = regsub(req.url, "^\/concept\/lists\/(.*)$", "/\1");
             set req.backend_hint = cm_concept_lists_api;
-    } elif (req.url ~ "^\/content\/query($|\/.*$)") {
-            set req.url = regsub(req.url, "^\/content\/query($|\/.*$)", "/query\1");
+    } elif (req.url ~ "^\/content\/query($|.*$)") {
+            set req.url = regsub(req.url, "^\/content\/query($|.*$)", "/query\1");
```

Comment:
**thought**: I'll contact you about a small change in the regex, I'm worried that this is a bit too permissive and for example something like /content/queryadasdasdasdasdadad matches and forwards to our service. I've written a more restrictive regex, using fragments of what you have previously suggested to me.